### ISSUE TITLE
[Feature request] speed cameras icon

### ISSUE THREAD

**Issue by katerynakocherhina** [osmandapp/Osmand#8793]
The speed cameras icons are smaller than the Parking lot icon. Both of them are blue which makes it difficult to distinguish the icons.

**Comment by chereskata** [comment]
I am working on a solution here: https://github.com/chereskata/osmandcar

The zoom limit of 15 is hardcoded i believe. But as the default.render.xml is 10k LOC i could have overseen it yet.

**Comment by vshcherb** [comment]
Yes icons are hard coded in style but also they are probalby not present in the map data for zooms <= 14 . https://github.com/osmandapp/OsmAnd-resources/blob/master/obf_creation/rendering_types.xml

**Comment by chereskata** [comment]
@vshcherb This xml is not user-adjustable is it?

**Comment by vshcherb** [comment]
nope

**Comment by vshcherb** [comment]
it's adjustable during map creation i.e. in OsmAndMapCreator
